android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/datatypes/RoofStatusTests.java

```diff
@@ -0,0 +1,73 @@
+package com.smartdevicelink.test.rpc.datatypes;
+
+import com.smartdevicelink.marshal.JsonRPCMarshaller;
+import com.smartdevicelink.proxy.rpc.Grid;
+import com.smartdevicelink.proxy.rpc.RoofStatus;
+import com.smartdevicelink.proxy.rpc.WindowState;
+import com.smartdevicelink.proxy.rpc.enums.DoorStatusType;
+import com.smartdevicelink.test.JsonUtils;
+import com.smartdevicelink.test.TestValues;
+import com.smartdevicelink.test.Validator;
+
+import junit.framework.TestCase;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Hashtable;
+import java.util.Iterator;
+
+public class RoofStatusTests extends TestCase {
+
+    private RoofStatus msg;
+
+    @Override
+    public void setUp() {
+        msg = new RoofStatus(TestValues.GENERAL_GRID, TestValues.GENERAL_DOOR_STATUS_TYPE);
+        msg.setState(TestValues.GENERAL_WINDOW_STATE);
+    }
+
+    public void testRpcValues() {
+        // Test Values
+        Grid location = msg.getLocation();
+        DoorStatusType status = msg.getStatus();
+
+        // Valid Tests
+        assertTrue(Validator.validateGrid(msg.getLocation(), location));
+        assertEquals(TestValues.MATCH, TestValues.GENERAL_DOOR_STATUS_TYPE, status);
+    }
```

Comment:
@kostyaBoss Below line 37 I would add this line `assertEquals(TestValues.MATCH, TestValues.GENERAL_WINDOW_STATE, windowState);`